cli/users.go

```diff
@@ -232,6 +232,46 @@ var cmdUsers = []cobra.Command{
 			logJSON(user)
 		},
 	},
+	{
+		Use:   "resetpasswordrequest <email>",
+		Short: "Reset password request",
```

Comment:
```suggestion
		Short: "Send reset password request",
```
Same as Long description